Types.ts

```diff
@@ -4,8 +4,22 @@ export type Session = {
     ApiGatewayVersion: string;
 }
 
+
 export type Metadata = {
     title: string;
     playbackUrl: string;
     posterImage: string;
-}
\ No newline at end of file
+}
+
+
+interface Errors {
+    [key: number]: string
+}
+
+export const Errors: Errors = {

```

Comment:
This is nice but can we use an Enum instead? Never used an enum in TypeScript so not sure if it can be abused as a dictionary.
https://www.typescriptlang.org/docs/handbook/enums.html